cmd/crc/cmd/daemon.go

```diff
@@ -182,26 +171,10 @@ func run(configuration *types.Configuration) error {
 	}()
 
 	go func() {
-		if runtime.GOOS == "darwin" {
-			for {
-				conn, err := vsockListener.Accept()
-				if err != nil {
-					log.Errorf("vpnkit accept error: %s", err)
-					continue
-				}
-				go func() {
-					if err := vn.AcceptVpnKit(conn); err != nil {
-						log.Errorf("vpnkit accept error: %s", err)
-
-					}
-				}()
-			}
-		} else {
-			mux := http.NewServeMux()
-			mux.Handle(types.ConnectPath, vn.Mux())
-			if err := http.Serve(vsockListener, mux); err != nil {
-				errCh <- errors.Wrap(err, "virtualnetwork http.Serve failed")
-			}
+		mux := http.NewServeMux()
```

Comment:
simplicity !

Reply:
In a way, yes, the VM networking is probably more complicated without vpnkit :)

Reply:
> networking is probably more complicated without vpnkit :)

can this be implemented?

Reply:
This looked doable using https://github.com/Code-Hex/vz/blob/a2ebc854a26128c56da3cbeea653eb73e6c145c8/network.go#L95-L101 but I did not try to implement it at all.

Reply:
Added.